src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -487,7 +487,7 @@
 (rf/reg-event-fx :wallet/stop-get-suggested-routes
  (fn []
    {:fx            [[:dispatch [:wallet/clean-routes-calculation]]]
-    :json-rpc/call [{:method   "wallet_stopSuggestedRoutesV2AsyncCalcualtion"
+    :json-rpc/call [{:method   "wallet_stopSuggestedRoutesAsyncCalcualtion"
```

Comment:
On the previous status-go version, there was a typo in the endpoint name (Calcualtion). Now is fixed on status-go side, so this should be `wallet_stopSuggestedRoutesAsyncCalculation`